main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 
 	"github.com/netlify/gotrue/cmd"
+	_ "github.com/netlify/gotrue/docs" // used by go-swagger to find docs
```

Comment:
should this point to /supabase/ ?

Reply:
hmm not too sure if it will break anyone self-hosting gotrue because that relies on the module name [here](https://github.com/supabase/gotrue/blob/9dfe6fe65e5607fbd020233c0bc98dd954f084a8/go.mod#L1), but i'm in favour of changing it too